CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+To submit pull requests for mx, you need to sign the [Oracle Contributor Agreement][1].
+There is a white-list for contributors who have signed the OCA so please add a comment
+to your first pull request indicating the name you used to sign the OCA if it isn't clear
+from your github profile.
+
+Pull requests can only be merged by committers and a committer cannot merge
+their own pull requests. 
```

Comment:
while I'm aware that "their" is acceptable grammar here, I think "his or her" reads better, especially for those whose first language is English ;-)